### DISCUSSION TITLE
无对话数限制使用ChatGPT GPT-4（要求Plus账号）

### DISCUSSION THREAD

**Discussion by ghost** [viopsa233/AI-testers-Forum#61] [交流]
就是用手机版的GPT-4模型
第一步，https://ai.com 官方登录
第二步，进入 http://chat.openai.com/api/auth/session 复制Access Token
第三步，进入 https://chat.zhile.io 输入Access Token登录，选择GPT-4(Mobile, V2)模型

**Comment by Adelinamelanie** [comment]
能否用這個做一個類似 一个简单的EdgeGPT网页前端——ChatSydney 這樣的前端呀，還可以更改對話的那種😁

**Reply by ghost** [reply]
https://github.com/acheong08/ChatGPT-to-API
配上 https://github.com/ztjhz/BetterChatGPT

**Reply by Adelinamelanie** [reply]
謝謝🚀

**Comment by Adelinamelanie** [comment]
我現在直接用手機ios端，選擇chat4對話一下，然後網頁端口就有無限次數版本的chat4模型了😁

**Comment by q2408808** [comment]
如果没有对话限制次数，那么合买gpt4就有可能了
